spk/homeassistant/src/dsm-control.sh

```diff
@@ -0,0 +1,89 @@
+#!/bin/sh
+
+# Package
+PACKAGE="homeassistant"
+DNAME="Home Assistant"
+
+# Others
+INSTALL_DIR="/usr/local/${PACKAGE}"
+PYTHON_DIR="/usr/local/python3"
+PYTHONPATH="${INSTALL_DIR}/share/${PACKAGE}"
```

Comment:
The description does not match the content here. Either correct that, or add this to PATH and use that in the start command.